src/components/tutorial-card/index.tsx

```diff
@@ -22,6 +26,7 @@ function TutorialCard({
 	hasVideo,
 	hasInteractiveLab,
 	eyebrowSlot,
+	eyebrowSlotAriaLabel = getSpeakableDuration(duration),
```

Comment:
We now support an `eyebrowSlotAriaLabel` prop. This allows consumers who are passing custom content into `eyebrowSlot` to hook into `build-aria-label` for the card, to ensure custom content is reflected in the `aria-label`.

This felt necessary to support including progress information in the `aria-label`.

We default to `getSpeakableDuration` if `eyebrowSlotAriaLabel` isn't provided, so that at minimum, we'll always have an `aria-label` equivalent to what we were generating previously.